src/commands/test.js

```diff
@@ -156,6 +159,16 @@ function getPlatform(logsOpt) {
   throw new Error(`Unsupported platform: ${type} ${arch} ${majorVersion}`)
 }
 
+async function getLinuxInfo() {
+  try {
+    let info = await system.run("cat /etc/*-release | egrep -e '(^VERSION|^NAME)='", {trim: true})
+    return info.replace(/[VERSION=]|[NAME=]|['"]+/g, '').split('\n')
```

Comment:
What do you think of returning those values as an object instead of an array? I think it will be clearer to understand what's each value (instead of relying on indexes).

```javascript
[ 'Ubuntu', '20.04.1 LT (Focal Fossa)' ]
```

vs
```javascript
{ name: 'Ubuntu', version: '20.04.1 LT (Focal Fossa)' }
```

Reply:
Actually thinking about it, I don't know if the order of those line will be always the same, maybe I should grep them separately